homeassistant/components/keymitt_ble/__init__.py

```diff
@@ -0,0 +1,54 @@
+"""Integration to integrate Keymitt BLE devices with Home Assistant."""
+from __future__ import annotations
+
+import logging
+from typing import TYPE_CHECKING
+
+from microbot import MicroBotApiClient
+
+from homeassistant.components import bluetooth
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_ACCESS_TOKEN, CONF_ADDRESS, Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryNotReady
+
+from .const import DOMAIN
+from .coordinator import MicroBotDataUpdateCoordinator
+
+if TYPE_CHECKING:
+    from bleak.backends.device import BLEDevice
+
+_LOGGER: logging.Logger = logging.getLogger(__package__)
+PLATFORMS: list[str] = [Platform.SWITCH]
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up this integration using UI."""
+    hass.data.setdefault(DOMAIN, {})
+    token: str = entry.data[CONF_ACCESS_TOKEN]
+    bdaddr: str = entry.data[CONF_ADDRESS]
+    ble_device: BLEDevice | None = bluetooth.async_ble_device_from_address(hass, bdaddr)
```

Comment:
`async_ble_device_from_address` is already typed so we can drop the typing here.

```suggestion
    ble_device = bluetooth.async_ble_device_from_address(hass, bdaddr)
```